example/test/_features/example_app/validation.feature

```diff
@@ -62,4 +62,57 @@ Feature: validations
     Then I should see 'Application sent' on the page
     Then I click the 'Start again' button
     Then I should be on the 'landing-page' page showing 'Choose one of the options below and press continue.'
-  
\ No newline at end of file
+    
+  @complex_form
+  Scenario: Full Complex Form Submission
+    Given I start the 'base' application journey
+    Then I continue to the next step
+    Then I should see the 'Select an option below and press continue' error
+    Then I choose 'Complex form'
+    Then I continue to the next step
+    Then I click the 'Continue' button
+    Then I should see the 'Enter your full name' error
+    Then I fill 'name' with 'Jane Doe'
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Enter your date of birth in the correct format; for example, 31 10 1990' error
+    Then I fill the date 'dateOfBirth' with '1-1-1899'
+    Then I click the 'Continue' button
+    Then I should see the 'Enter a date after 1 1 1900' error
+    Then I fill the date 'dateOfBirth' with '1-1-3000'
+    Then I click the 'Continue' button
+    Then I should see the 'Enter a date that is in the past' error
+    Then I enter a date of birth for a 30 year old
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Enter details of your building and street' error
+    Then I should see the 'Enter a town or city' error
+    Then I should see the 'Enter your postcode' error
+    Then I fill 'building' with '10 Downing Street'
+    Then I fill 'townOrCity' with 'London3'
+    Then I fill 'postcode' with 'W12 3DE'
+    Then I click the 'Continue' button
+    Then I should see the 'Enter a town or city without including digits' error
+    Then I fill 'townOrCity' with 'London'
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Select all options that apply to you.' error
+    Then I select 'Salary' 
+    Then I select 'Child Benefit' 
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Select where the appeal hearing is to be held' error
+    Then I select 'England and Wales'
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Enter details about why you are making a complaint' error
+    Then I fill 'complaintDetails' text area with 'I would like to make a complaint'
+    Then I click the 'Continue' button
+    Then I click the 'Continue' button
+    Then I should see the 'Select an appeal stage from the list' error
+    Then I select field 'appealStages' and value '01. First Tier IAC Appeal - In Country Appeals'
```

Comment:
Alter line 113 to read as 'Then I select '01. First Tier IAC Appeal - In Country Appeals' from 'appealStages'. (note I have changed the word 'and' to 'from' as well)